docs/src/ex/0_Introduction.md

```diff
@@ -0,0 +1,178 @@
+
+```julia
+# load Turing and MCMCChain
+using Turing, MCMCChain
+
+# load the distributions library
+using Distributions
+
+# load stats plots for density plots
+using StatPlots
+```
+
+First, we will define the coin-flip model using Turing.
+
+```julia
+@model coinflip(y) = begin
+
+    # our prior belief about the probability of heads in a coin
+    p ~ Beta(1, 1)
+
+    # the number of observations
+    N = length(y)
+    for n in 1:N
+        # heads or tails of a coin are drawn from a Bernoulli distribution
+        y[n] ~ Bernoulli(p)
+    end
+end;
+```
+
+After defining the model, we can approximate the posterior distribution by pulling samples from the distribution. In this example, we use a [Hamiltonian Monte Carlo](https://en.wikipedia.org/wiki/Hamiltonian_Monte_Carlo) sampler to construct these samples. Later tutorials will give more information on the samplers available in Turing and discuss their use for different models.
+
+```julia
+# setting of Hamiltonian Monte Carlo (HMC) sampler
+iterations = 1000
+ϵ = 0.05
+τ = 10
+
+# start sampling
+chain = sample(coinflip(data), HMC(iterations, ϵ, τ));
+```
+
+After finishing the sampling process, we can visualize the posterior distribution approximated using Turing against the posterior distribution in closed-form. We can extract the chain data from the sampler using the `Chains(chain[:p])` function. This contains all the values of `p` we drew while sampling.
+
+```julia
+# construct summary of the sampling process for the parameter p, i.e. the probability of heads in a coin
+p_summary = Chains(chain[:p])
+```
+
+```
+Object of type "Chains"
+
+Iterations = 1:1000
+Thinning interval = 1
+Chains = 1
+Samples per chain = 1000
+
+[0.859911; 0.219831; … ; 0.496273; 0.473286]
+```
+
+Now we can build our plot:
+
+```julia
+# compute the posterior distribution in closed-form
+N = length(data)
+heads = sum(data)
+updated_belief = Beta(prior_belief.α + heads, prior_belief.β + N - heads)
+
+# visualize a blue density plot of the approximate posterior distribution using HMC (see Chain 1 in the legend)
+p = densityplot(p_summary, xlim = (0,1), legend = :best, w = 2, c = :blue)
+
+# visualize a green density plot of posterior distribution in closed-form
+plot!(p, range(0, stop = 1, length = 100), pdf.(Ref(updated_belief), range(0, stop = 1, length = 100)),
+        xlabel = "probability of heads", ylabel = "", title = "", xlim = (0,1), label = "Closed-form",
+        fill=0, α=0.3, w=3, c = :lightgreen)
+
+# visualize the true probability of heads in red
+vline!(p, [p_true], label = "True probability", c = :red);
+```
+
+![sdf](https://user-images.githubusercontent.com/7974003/44995682-25477880-af9c-11e8-850b-36e4b6d756ea.png)
+
+As we can see, the Turing model closely approximates the true probability. Hopefully this has provided an introduction to Turing's simpler applications. More advanced usage is demonstrated in later tutorials.
```

Comment:
Maybe put the latter two sentences in a separate paragraph. Also `Hopefully this has` -> `Hopefully this tutorial`, `introduction` -> `easy-to-follow, yet informative introduction`, and `is demonstrated -> will be demonstrated`.